Source: Dan Callaghan <djc@djc.id.au>
Reason: we handle icon cache updates outside of the build
Upstream: not yet

diff -ur mail-notification-5.4/jbsrc/jb.c mail-notification-5.4-hacked/jbsrc/jb.c
--- mail-notification-5.4/jbsrc/jb.c	2008-05-23 01:47:04.000000000 +1000
+++ mail-notification-5.4-hacked/jbsrc/jb.c	2010-10-09 14:01:58.038290732 +1000
@@ -156,6 +156,7 @@
   jb_variable_add_bool("debug", NULL, NULL, 0, FALSE);
   jb_variable_add_bool("regression-tests", NULL, NULL, 0, FALSE);
   jb_variable_add_bool("gconf-sanity-check", NULL, NULL, JB_VARIABLE_C_DEFINE, TRUE);
+  jb_variable_add_bool("update-gtk-icon-cache", NULL, NULL, 0, TRUE);
 }
 
 void
@@ -364,10 +365,12 @@
   if (jb_variable_get_bool("hotmail"))
     jb_group_add_data_file(group, "hotmail.png", "$pkgdatadir");
 
-  rule = jb_rule_new();
-  jb_rule_set_install_message(rule, "updating the GTK+ icon cache");
-  jb_rule_add_install_command(rule, "-gtk-update-icon-cache -f -t $datadir/icons/hicolor");
-  jb_group_add_resource(group, JB_GROUP_RESOURCE(rule));
+  if (jb_variable_get_bool("update-gtk-icon-cache")) {
+    rule = jb_rule_new();
+    jb_rule_set_install_message(rule, "updating the GTK+ icon cache");
+    jb_rule_add_install_command(rule, "-gtk-update-icon-cache -f -t $datadir/icons/hicolor");
+    jb_group_add_resource(group, JB_GROUP_RESOURCE(rule));
+  }
 
   jb_group_add(group);
 
